scripts/microbe_stage/setup.as

```diff
@@ -363,6 +363,20 @@ void cellHitAgent(GameWorld@ world, ObjectID firstEntity, ObjectID secondEntity)
 
 }
 
+uint getHexCountOrganelle(PlacedOrganelle@ organelle)
```

Comment:
I think this shouldn't be a separate function. AngelScript doesn't have an optimizer so this is going to hit the performance hard being in a critical physics loop.